doc/03-Configuration.md

```diff
@@ -128,6 +128,16 @@ the cluster nodes resynchronise their states after each executed query, and may
 However, this does not necessarily have to be the case if, for instance, Icinga DB is only allowed to connect to a
 single cluster node at a time. This is the case when a load balancer does not randomly route connections to all the
 nodes evenly, but always to the same node until it fails, or if your database cluster nodes have a virtual IP address
-fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the 
+fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the
 `/etc/icingadb/config.yml` file to disable it entirely, as Icinga DB doesn't have to wait for cluster
 synchronisation then.
+
+### Systemd Journald Fields
+
+Using the `systemd-journald` logging output results in sending structured log entries to journald,
+including fields with additional information that would be included in the `console` logging output mode.
```

Comment:
I'm not sure about the level of detail here. I don't know if the average user would understand what "including fields with additional information that would be included in the `console` logging output mode" is referring to. I think I'd omit this paragraph and extend the following one a bit mentioning that the messages logged to journald can contain extra `ICINGADB_*` fields that are not shown by default.

Reply:
I tried to rephrase this section. Please take another look.